cabal-testsuite/PackageTests/ExtraProgPath/setup.test.hs

```diff
@@ -0,0 +1,7 @@
+import Test.Cabal.Prelude
+
+-- Test that setup shows all the 'autogen-modules' warnings.
+main = cabalTest $ do
+  skipIfWindows
```

Comment:
The GitHub virtual environments usually have the Git Bash installed, so maybe this could also run under Windows.